keyvalue/server/pom.xml

```diff
@@ -46,7 +46,12 @@
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-server</artifactId>
+      <artifactId>alluxio-core-server-master</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-worker</artifactId>
```

Comment:
It may make less sense for internal dependencies. For external dependencies, we want to avoid the case where we remove a dependency and the lose a transitive dependency we were implicitly relying on. @apc999 do you have any thoughts on this for internal dependencies?